### PR TITLE
Validate image message without image array (width == 0 and height == 0)

### PR DESCRIPTION
**Pull request by wkentaro**
issue: #771

### CODE REVIEW

**Comment by k-okada** [comment]
I think it is better to call ROS_WARN or ROS_ERROR to notifiy the input image is not valid

**Comment by YoheiKakiuchi** [comment]
Which node can publish invalid image message ?

File: jsk_perception/src/rgb_decomposer.cpp [hunk 1]
```diff
@@ -64,6 +64,9 @@ namespace jsk_perception
   {
     cv_bridge::CvImagePtr cv_ptr = cv_bridge::toCvCopy(
       image_msg, image_msg->encoding);
+    if ((image_msg->width == 0) && (image_msg->height == 0)) {
```
Comment (garaemon) [inline jsk_perception/src/rgb_decomposer.cpp]: check before toCvCopy

**Comment by wkentaro** [comment]
> Which node can publish invalid image message ?

`/apply_mask` node does publish. Please see the issue: #771 .

**Comment by wkentaro** [comment]
> I think it is better to call ROS_WARN or ROS_ERROR to notifiy the input image is not valid

I added https://github.com/jsk-ros-pkg/jsk_recognition/pull/772/files#diff-02d617fc79be929b48d2aaf83128747dR66 .

> check before toCvCopy

I changed.

**Pull request merged**